.github/workflows/build.yml

```diff
@@ -66,17 +66,31 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - run: ./scripts/ci-select-xcode.sh 16.3
+      - run: brew install xcodegen
+        if: ${{ contains(matrix.scheme, 'SessionReplay-CameraTest') }}
+      - run: make session-replay-camera-test-xcode
+        if: ${{ contains(matrix.scheme, 'SessionReplay-CameraTest') }}
```

Comment:
`m`: It might make sense to add a flag `generate` to the matrix and set it to `true` only for the `SessionReplay-CameraTest`, instead of doing a name-based check to keep the complexity lower

Reply:
i thought about doing this, but this is just a temporary measure while we have a mix of xcodegen-managed projects and otherwise. once they're all across the finish line, this will disappear anyways.